apps/src/applab/applab.js

```diff
@@ -894,6 +912,24 @@ Applab.init = function(config) {
 };
 
 /**
+ * Subscribe to state changes on the store.
+ * @param {!Store} store
+ */
+function subscribeToRedux(store) {
```

Comment:
Based on the name, I was initially expecting this to be a method I call when I want to set up a new subscriber. In fact, this is a method that sets up some subscribers for us, and should never be called more than once. What about something like `setupReduxSubscribers` or `configureReduxSubscribers` (or maybe subscriptions instead of subscribers?)